ipalib/install/certmonger.py

```diff
@@ -462,6 +463,10 @@ def request_cert(
         request_parameters['ca-profile'] = profile
     if nss_user:
         request_parameters['nss-user'] = nss_user
+    if keysize is None:
```

Comment:
What if someone sets `keysize` and not `keytype`? Shouldn't both cases he handled?

Reply:
We literally will need to support more than RSA very soon. We can test EC already, though.

Reply:
I agree with @rcritten about the placeholders. It's weird that some algorithms are invalid and some not supported yet.